linode/stackscript/framework_resource_schema.go

```diff
@@ -37,6 +39,9 @@ var frameworkResourceSchema = schema.Schema{
 		"label": schema.StringAttribute{
 			Description: "The StackScript's label is for display purposes only.",
 			Required:    true,
+			Validators: []validator.String{
+				stringvalidator.LengthBetween(3, 128),
+			},
```

Comment:
Just double checking that there are no special validation rules for StackScript and Token labels?

Reply:
Not that I can see, [stackscripts](https://bits.linode.com/LinodeAPI/apinext/blob/d425f85958d5077f4890b53a64cdfe083c6b1c2b/apinext/maps/stackscript.py) only have a lengthValidator and same goes for [tokens](https://bits.linode.com/LinodeAPI/apinext/blob/d425f85958d5077f4890b53a64cdfe083c6b1c2b/apinext/maps/oauthtoken.py)